osf/management/commands/archive_registrations_on_IA.py

```diff
@@ -0,0 +1,66 @@
+import logging
+import django
+django.setup()
+
+import time
+from framework.celery_tasks import app as celery_app
+from osf.models import Registration
+from website import settings
+from osf.utils.requests import requests_retry_session
+
+logger = logging.getLogger(__name__)
+django.setup()
+
+from django.core.management.base import BaseCommand
+
+
+@celery_app.task(name='osf.management.commands.archive_registrations_on_IA')
+def archive_registrations_on_IA(dry_run=False, batch_size=100, guids=None):
+    if guids:
+        registrations = Registration.objects.filter(guids___id__in=guids)
+    else:
+        # randomize order so large registrations won't block all pigeon workers,
+        # and stuck registrations won't block repeatedly
+        registrations = Registration.find_ia_backlog().order_by('?')[:batch_size]
```

Comment:
Yeah I see that this is an improvement, but it would go better in the metadata check follow up, if we are repeatedly getting stuck will know what kind of detail we need in the placeholder values, potentially it's better from my point of view to use system tags on those registration, as is customary for weird registrations, basic the placeholder should be enough info to id a pigeon side spontanous failure from a irregular registration failure.

Reply:
Should also be able to skip "stuck" registrations, either via exclusion in `find_ia_backlog` or an `if registration.archiving: continue` below.